adage/analyze/api.py

```diff
@@ -342,7 +350,7 @@ def apply_filters(self, request, applicable_filters):
 
 class ParticipationResource(ModelResource):
     node = fields.IntegerField(attribute='node_id', null=False)
```

Comment:
@dongbohu : this rationale makes total sense to me. Can you put in a quick comment noting why there is a difference? Basically mentioning the performance and that we don't, at this time, intend to do a query that returns all nodes given an input gene.